### PR TITLE
Create embeddable entity for storing common file information

### PR DESCRIPTION
**Pull request by Koc**
Another one small step to reduce boilerplate when using the bundle.

TODO:

- [x] update documentation

### CODE REVIEW

**Comment by Koc** [comment]
@garak ready

File: Entity/File.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+     * @ORM\Column(name="mime_type", nullable=true)
+     */
+    protected $mimeType;
+
+    /**
+     * @ORM\Column(name="size", type="integer", nullable=true)
+     */
+    protected $size;
+
+    public function getName()
```
Comment (garak) [inline Entity/File.php]: What about some phpdoc here? And on other methods, too

Comment (Koc) [inline Entity/File.php]: I can add phpdocs with param/return types, but I haven't see any profit from method descriptions like `Sets the original file name`. It clear from method name what it do :)

Comment (Koc) [inline Entity/File.php]: @garak done

**Comment by Koc** [comment]
@garak yes, I'l do it later when other codebase will be moved to scalar type hints. Currently we are using this bundle on php 5.5.9 🤣

**Pull request merged**